src/tracker/tracker-funcs.jl

```diff
@@ -507,13 +507,13 @@ Convert the centroid coordinates from row and column to latitude and longitude d
 """
 function convertcentroid!(propdf, latlondata, colstodrop)
     latitude, longitude = [
-        [latlondata[c][x, y] for
+        [latlondata[c][Integer(round(x)), Integer(round(y))] for
```

Comment:
Note that this Integer cast is required for the lat-lon data which comes from EBSEG – it produces centroids with subpixel precision.
It think it would make more sense to do a proper affine transformation here, rather than getting a mapping of Xs, Ys to latitudes and longitudes. But that would be a thing for later, once we have a Julia-specific wrapper for PROJ in the project and we're not using python.

Reply:
You should use `round` with some care in Julia:
```julia
julia> Int(round(6.5))
6

julia> Int(round(7.5))
8
```
If you'd like to round the "usual" way with the cutoff at 0.5, you should use `RoundNearestTiesUp` as the rounding mode. https://github.com/WilhelmusLab/IceFloeTracker.jl/blob/ae20877a5833509afd8a356aa457059906830994/src/histogram_equalization.jl#L2)

Reply:
Thanks for the tip.

I think the rounding to the nearest even number in case of a tie is fine in this case – the numbers are floats, so the likelihood that we're going to be on precisely 0.5 is slim. 

If there's some situation where halves come out of the centroiding algorithm regularly, then reducing the bias by rounding to the nearest even number will probably be what we want.